javascript-practice/src/js/utils/api.js

```diff
@@ -21,6 +21,36 @@ const APIHandler = {
     } catch (error) {
       console.error(error);
 
+      showToastify(error.message, 'toastify-danger');
+    }
+  },
+
+  /**
+   * Sends a POST request to the endpoint with the provided product data
+   * @param {string} endpoint - The endpoint to which the request should be sent
+   * @param {Object} product - The product data to be sent
+   */
+  async post(endpoint, product) {
```

Comment:
Don't use the argument name 'product' here because it is a common method; we'll use it for other objects.